### PR TITLE
Fix backend-frontend communication

### PR DESCRIPTION
**Pull request by jamesreaver**
Klee outcome are now displayed on the website (was failing due to a deprecated call to Pusher's trigger function). Added Django timezone settings.

### CODE REVIEW

**Comment by jamesreaver** [comment]
We are working on fixing the failing tests.

**Comment by andronat** [comment]
@ccadar So we do want to keep Pusher eventually?

File: src/klee_web/settings.py [hunk 1]
```diff
@@ -88,6 +88,8 @@
 
 LANGUAGE_CODE = 'en-us'
 
+TIME_ZONE = 'Europe/London'
```
Comment (andronat) [inline src/klee_web/settings.py]: I think this is not a good idea as we are not the only people that might use this service. Is there any way to get the time zone from the system? Any Bash variables?

**Comment by helicopter88** [comment]
@andronat @ccadar we have to get rid of pusher, or make its secret keys public, if we want to get a fully working public repo.
Setting up a websocket server is not too hard, but it may require some extensive editing in both the backend and the frontend

**Comment by andronat** [comment]
I would suggest to dump Pusher and replace it with a time interval in AngularJS. If we want something more sophisticated we can revisit this in the future. BTW let's also wait for @ccadar opinion.

To be noted the time interval I'm talking about is not to refresh the webpage of the user!! This is important. Angular must request for new data behind the scenes!

@helicopter88 please get rid of the merge commit by rebasing your last 2 commits on top of @giacomoguerci initial PR.

File: src/klee_web/settings.py [hunk 2]
```diff
@@ -88,6 +88,8 @@
 
 LANGUAGE_CODE = 'en-us'
 
+TIME_ZONE = os.environ.get("TZ")
```
Comment (andronat) [inline src/klee_web/settings.py]: `/etc/timezone` is Ubuntu/Debian(?) specific solution and this file might not exist in the future.

As this is a python script you can do [this](http://stackoverflow.com/a/1111079).

**Comment by ains** [comment]
fwiw, I was going to switch from Pusher to Slanger (https://github.com/stevegraham/slanger), which is self hosted but API compatible, then you can have dummy keys for dev and secret keys for prod. 

That's if you still want to keep the real time stuff

**Comment by helicopter88** [comment]
That seems reasonable and shouldn't require any big change in the code too

**Comment by ccadar** [comment]
Thanks for the suggestion, @ains .  

I think we're ready to merge this, but I would first merge the commits concerning the timezone.

**Pull request merged**